### DISCUSSION TITLE
Add a way to have receivers that only care about the latest received value

### DISCUSSION THREAD

**Discussion by llucax** [frequenz-floss/frequenz-channels-python#339] [Ideas]
This means not logging when the buffer is full.

We have a proposal:

* https://github.com/frequenz-floss/frequenz-channels-python/pull/329

My feeling is this use case is special enough that it probably deserves a special receiver. `Broadcast` channel receivers are actually buffered, and if we want to receive only the latest value, that means that's unbuffered, so one option would be to add a `new_unbuffered_receiver()` that doesn't take a `limit` argument at all.

**Comment by Marenz** [comment]
That suggestion sounds fine to me
